### ISSUE TITLE
ansible-tower-cli doesn't trust system CA certs

### ISSUE THREAD

**Issue by willthames** [ansible/tower-cli#200]
It seems that ansible-tower-cli uses request's cacert.pem.

However, if you have a certificate signed by an internal CA, that's unlikely to be in that bundle. The standard practice for the majority of apps is to trust the system CA (curl, wget, etc. all manage this). 

You can override request's CA bundle using REQUEST_CA_BUNDLE, but need a way to allow that to be configurable as part of the ansible-tower-cli app.

Our previous tower cli install works fine (ansible-tower-cli 2.3.1 with requests 2.10.0) but the current tower cli install doesn't (ansible-tower-cli 3.0.0 with requests 2.11.1) - it is possible that someone did something manual to get 2.3.1 to work, however.

**Comment by matburt** [comment]
I reckon we could add this as a command line option and then use the `verify` parameter if provided:

```
requests.get('https://towerhost.com', verify='/path/to/certfile')
```

**Comment by willthames** [comment]
Would be good to have it configurable in tower_cli.cfg too.

**Comment by AlanCoding** [comment]
I agree with @willthames on that point. It should be a normal configuration variable. I could even see us abusing the `verify_ssl` parameter to allow a string, and pass into the call in the manner Matt suggested above. Either way, my expectation would that it is configurable, and will be visible in the config file.

**Comment by jangsutsr** [comment]
@willthames Custom certificate support is implemented, will come out with 3.0.1
